swagger-parser/src/main/scala/de/zalando/swagger/ValidationsConverter.scala

```diff
@@ -27,7 +30,8 @@ object ValidationsConverter {
     { case v: EnumValidation[_] => toEnumValidations(v) },
     { case v: StringValidation => toStringValidations(v) },
     { case n: NumberValidation[_] => toNumberValidations(n) },
-    { case o: ObjectValidation => toObjectValidations(o) }
+    { case o: ObjectValidation => toObjectValidations(o) },
+    { case p: QueryParameter[_] => toQueryParameterValidations(p) }
```

Comment:
This part looks strange, all other matchers check `Validation`, but this one works with `Parameter`. Maybe we could have something like `RequiredValidation`?